appserver/payara-appserver-modules/microprofile/metrics/src/main/java/fish/payara/microprofile/metrics/jmx/MBeanMetadata.java

```diff
@@ -259,16 +233,43 @@ public MetricType getTypeRaw() {
     public boolean isReusable() {
         return reusable;
     }
-    
+
     public void setTags(List<XmlTag> tags) {
         this.tags = tags;
     }
-    
+
     public void addTags(List<XmlTag> tags) {
         if (this.tags == null) {
             this.tags = new ArrayList<>();
         }
         this.tags.addAll(tags);
     }
 
+    @Override
```

Comment:
NB. Both methods are copied straight from `DefaultMetadata` so they follow the same logic. This is needed because the registry checks for illegal reuse comparing _new_ and _existing_ `Metadata` with `equals`. Not having it implemented fails the test and considers them _not equal_ and therefore illegal for reuse. It is beyond me how this did not pop up earlier.